sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableAsyncClient.java

```diff
@@ -845,41 +842,47 @@ public void close() {
      * Retrieves details about any stored access policies specified on the table that may be used with Shared Access
      * Signatures.
      *
-     * @return A paged reactive result containing the HTTP response and the table's
+     * @return A reactive result containing the table's {@link TableSignedIdentifier access policies}.
+     */
+    @ServiceMethod(returns = ReturnType.COLLECTION)
+    public Mono<List<TableSignedIdentifier>> listAccessPolicies() {
```

Comment:
Instead of returning a Mono<List<T>> use a container like storage does - `public Mono<BlobContainerAccessPolicies> getAccessPolicy()`.